cookomatic_api/tests/api/test_dish.py

```diff
@@ -18,6 +19,23 @@ def setUp(self):
         self.testbed.init_memcache_stub()
         ndb.get_context().clear_cache()
 
+        self.SAMPLE_DISH = {
```

Comment:
pylint is [complaining](https://travis-ci.org/cookomatic/cookomatic-api/jobs/206493463#L707) that this variable should be lowercase